interface/forms/gad7/gad7.inc.php

```diff
@@ -14,6 +14,8 @@
 
  require_once("../../globals.php");
  require_once("$srcdir/api.inc");
+ require_once("$srcdir/patient.inc");
+ require_once("$srcdir/globals.inc.php");
```

Comment:
@ruthkonyn Here is your issue with unwanted event. Why do you need this script? I can't think of a reason.
Remove....

Reply:
@ruthkonyn Did you also remove this? If not, do. you don't need.

Reply:
also I have fix in next patch. look here if you want to try mod yourself: https://github.com/openemr/openemr/pull/4459